python/tests/src/tests/kat/test_ambassador.py

```diff
@@ -0,0 +1,15 @@
+# Import all the real tests from other files, to make it easier to pick and choose during development.
+
```

Comment:
The comment should go if all the imports have gone.

IIRC the point of all this stuff had to do with the way `pytest` found tests -- I assume that's different now...